contrib/NeonVM/neonvm.c

```diff
@@ -277,24 +277,31 @@ void exec_rtl_raiseException(TExecutor *self, const char *name, const char *info
     string_freeString(i);
 }
 
-static uint32_t exec_getOperand(TExecutor *self)
+static unsigned int exec_getOperand(TExecutor *self)
 {
-    uint32_t r = (self->object->code[self->ip+1] << 24) | (self->object->code[self->ip+2] << 16) | (self->object->code[self->ip+3] << 8) | self->object->code[self->ip+4];
-    self->ip += 5;
+    unsigned int r = 0;
+    while (self->ip < self->object->codelen) {
```

Comment:
You should be able to call the existing `get_vint` function here instead of copying the implementation.